### DISCUSSION TITLE
Documentation to code

### DISCUSSION THREAD

**Discussion by codeinflsk** [pallets/flask#4357] [Q&A]
May I know how to add documentation to your code

**Comment by tachyondecay** [comment]
The documentation is written using [Sphinx](https://www.sphinx-doc.org/en/stable/) and stored in the `docs` directory of the repo. If you’re thinking of improving the documentation, please review the [contributing guidelines](https://github.com/pallets/flask/blob/main/CONTRIBUTING.rst) before you submit any PRs.

**Comment by codeinflsk** [comment]
Thank you for your response.
